cmd/api/main.go

```diff
@@ -208,6 +209,24 @@ func main() {
 		go backupScheduler.Run()
 	}
 
+	var metricsPublisher *publisher.Publisher
+	if config.TelemetryPublisher.Enabled {
+		nodeID, err := chainStacks[config.Chains[0].ChainID].Store.GetID(context.Background())
```

Comment:
Nit: WDYT of doing the `GetID()` call before the L213 `if`, and always print it as `log.Info()`?
I think that might be useful in general to identify logs?

Reply:
Sure!